.travis.yml

```diff
@@ -2,10 +2,18 @@ language: c++
 sudo: false
 env:
   - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
   - NODE_VERSION="0.12"
+  - NODE_VERSION="iojs-v1"
```

Comment:
In that case, I'd rather stick with supporting `v0.12`, `v4`, and `v5` in that case. It's essentially the same as what we're testing on now (minus `v0.10`), and we don't have to worry about all the differences iojs might have going forward.